package.json

```diff
@@ -1,20 +1,22 @@
 {
   "name": "@hireupau/hucssley",
-  "version": "1.0.0",
+  "version": "1.0.2-sass.1",
   "description": "",
   "keywords": [],
-  "main": "src/_index.scss",
+  "main": "dist/index.css",
```

Comment:
I should double check import usage as this change will change the result of doing:
`import huccssley from '@hireupau/hucssley'`